src/rust/Dockerfile

```diff
@@ -425,6 +425,15 @@ FROM rust-dist AS osquery-generator-deploy
 COPY --from=build /outputs/osquery-generator /
 ENTRYPOINT ["/osquery-generator"]
 
+##### generator-executor
+FROM rust-dist AS generator-executor-deploy
```

Comment:
should this be "plugin-executor-deploy"?

Reply:
No, although this is up for reexamination later. If you look at Cargo.toml this package outputs 2 bins. The hope is to share logic between the two in this one package's lib.